src/test/java/com/segment/analytics/android/integrations/taplytics/TaplyticsTest.java

```diff
@@ -75,14 +73,12 @@
   @Test public void initialize() {
     ValueMap settings = new ValueMap() //
         .putValue("apiKey", "foo") //
-        .putValue("sessionMinutes", 20)
-        .putValue("liveUpdate_v2", "true") //
-        .putValue("shakeMenu_v2", "false")
-        .putValue("turnMenu_v2", "default");
+        .putValue("sessionMinutes", 20).putValue("liveUpdate_v2", "true") //
```

Comment:
Yeah IntelliJ isn't the smartest when it comes to unwrapping these. That's why we've been migrating to https://github.com/f2prateek/gradle-android-javafmt-plugin, but will take us some time to get all the repos.